op-node/p2p/store/scorebook.go

```diff
@@ -23,11 +24,11 @@ type scoreRecord struct {
 }
 
 func (s *scoreRecord) SetLastUpdated(t time.Time) {
-	s.LastUpdate = t.Unix()
+	atomic.StoreInt64(&s.LastUpdate, t.Unix())
 }
 
 func (s *scoreRecord) LastUpdated() time.Time {
-	return time.Unix(s.LastUpdate, 0)
+	return time.Unix(0, atomic.LoadInt64(&s.LastUpdate))
```

Comment:
You are swapping the seconds and nanoseconds arguments here, while the `t.Unix()` is still returning a timestamp in seconds.
Also, I recommend you change the struct field to `atomic.Int64`, to ensure atomic access and avoid any field-alignment issues.

Reply:
@protolambda this should be resolved now, thanks for the fast feeback.

Reply:
What I meant is to change the

```
LastUpdate int64      `json:"lastUpdate"` 
```
line in the struct definition to
```
LastUpdate atomic.Int64      `json:"lastUpdate"`
```

The type has helper methods to load/store, so the user doesn't have to remember to use the atomic load/store functions with the integer.

And formatting the file imports for the native libraries to be listed together would be good too. (think the linter in CI may pick up on that, but takes a while to run)

Reply:
Also, can you squash the commits? To keep the change atomic (no pun intended :sweat_smile: )

Reply:
updated